src/Course/List.hs

```diff
@@ -323,7 +323,7 @@ produce f x =
 -- >>> notReverse Nil
 -- []
 --
--- prop> \x y -> let types = x :: List Int in notReverse x ++ notReverse y == notReverse (y ++ x)
+-- prop> \x y -> let types = x :: List Int in notReverse x ++ notReverse y == notReverse (x ++ y)
```

Comment:
This seems more correct to me (please let me know if I'm off-base here!)

Reply:
You are mistaken. Try it!

Reply:
Ah - I was under the impression that the solution would look like:
```.hs
notReverse :: List a -> List a
notReverse = id
```

But after looking at the solution, I see my mistake. Thanks!

Reply:
(Should have looked at the solutions first... lol)